core/sql/sqlcomp/CmpSeabaseDDLtable.cpp

```diff
@@ -7791,7 +7791,7 @@ void CmpSeabaseDDL::alterSeabaseTableAddPKeyConstraint(
   // this new primary key.
   // Do this optimization in mode_special_4 only.
   Lng32 len = 0;
-  Int64 rowCount = 0;
+  Lng32 rowCount = 0;
```

Comment:
Same comment as above. The container is not large enough to hold the return value from executeImmediate() . And since it's on stack it can corrupt another value on the stack.